examples/1-getting-started.ipynb

```diff
@@ -0,0 +1,322 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with steps\n",
+    "\n",
+    "This notebook shows how to **create** steps, **fit** them to data, **transform** new data and take advantage of persistence"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "from steps.base import Step, BaseTransformer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grabbing some data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll import a dataset from scikit-learn for our experiments and divide it into training and test sets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import load_digits\n",
+    "\n",
+    "digits = load_digits()\n",
+    "X_digits, y_digits = digits.data, digits.target\n",
+    "\n",
+    "num_train = int(0.8 * len(y_digits))\n",
```

Comment:
@mromaniukcdl I usually take advantage of sklearn.model_selection module with either simple `train_test_split` (as here) or `StratifiedKFold` . Is there any reason not to go with 
`X_train, X_valid, y_train, y_valid = train_test_split(X_digits, y_digits, test_size=0.2)` ?

Reply:
I was trying to do something along the lines of this example: http://scikit-learn.org/stable/auto_examples/exercises/plot_digits_classification_exercise.html#sphx-glr-auto-examples-exercises-plot-digits-classification-exercise-py

StratifiedKFold would be better in principle although a few more lines of code. But we could switch to it.

Reply:
+1